### PR TITLE
Maintain a equal distribution of instances across AZs in the same region.

### PR DESCRIPTION
**Pull request by jeanschmidt**

### CODE REVIEW

File: terraform-aws-github-runner/modules/runners/scale-down.tf
Comment (DanilBaibak) [inline terraform-aws-github-runner/modules/runners/scale-down.tf:47]: Could you pls run `terraform fmt -recursive`

**Review by DanilBaibak — APPROVED** [review]
LGTM! Would be great if you could test you changes in the canary environment before merging.

File: terraform-aws-github-runner/modules/runners/scale-down.tf
Comment (jeanschmidt) [reply terraform-aws-github-runner/modules/runners/scale-down.tf:47]: Sure!

This will trigger LOTS of unrelated chages, but this is OK :)

**Pull request merged**
